packages/core/admin/ee/admin/pages/SettingsPage/pages/ApplicationInfosPage/components/AdminSeatInfo/index.js

```diff
@@ -39,34 +40,44 @@ const AdminSeatInfo = () => {
             textColor={licenseLimitStatus === 'OVER_LIMIT' ? 'danger500' : ''}
             fontWeight={licenseLimitStatus === 'OVER_LIMIT' ? 'bold' : ''}
           >
-            {currentUserCount || 'NA'}
+            {formatMessage(
+              {
+                id: 'Settings.application.ee.admin-seats.count',
+                defaultMessage: '{currentUserCount}/{permittedSeats}',
```

Comment:
@ivanThePleasant In case the `NA` case needs to be handled, you could do that the following way:

```suggestion
                defaultMessage: '{currentUserCount, select, undefined {NA} other {{currentUserCount}}}/{permittedSeats}',
```

Reply:
This is actually redundant atm because if there is no license info, the component which uses these messages will not be shown